internal/convert.go

```diff
@@ -373,38 +419,66 @@ func (conv *Conv) getStatementStat(s string) *statementStat {
 	return conv.stats.statement[s]
 }
 
-func (conv *Conv) skipStatement(l []nodes.Node) {
-	if conv.schemaMode() { // Record statement stats on first pass only.
+func (conv *Conv) SkipStatement(l []nodes.Node) {
+	if conv.SchemaMode() { // Record statement stats on first pass only.
 		s := prNodes(l)
 		VerbosePrintf("Skipping statement: %s\n", s)
 		conv.getStatementStat(s).skip++
 	}
 }
+func (conv *Conv) SkipStatementMySQL(stmtType string) {
```

Comment:
Is it possible to merge `SkipStatement` and `SkipStatementMySQL`? What I mean is to move `s := prNodes(l)` out from this function and then `SkipStatement` should work for both `Postgres` and `Mysql`. Same for `ErrorInStatement ` and `ErrorInStatementMYSQL`. 

If this has been done in next PRs, then I am fine with this right now.

Reply:
I didn't merge it at first because it needed changes in pgdump. But now after your suggestion, I have merged these functions in latest commit.